provisioning/miniprov/test/test_cdf.py

```diff
@@ -221,6 +221,149 @@ def ret_values(value: str) -> Any:
         self.assertEqual(1, len(ret[0].pools.value))
         self.assertEqual(Text('StorageSet-1__sns'), ret[0].pools.value[0])
 
+    def test_allowed_failure_generation_4_2_0(self):
```

Comment:
@SwapnilGaonkar7 , This is good but instead of 2 functions, one for each layout (4+2+0 and 4+2+2), I think we can have just one function and pass layout params to it. what do you think?

Reply:
It would be great if we can run the function and verify the allowed_failures vector for all the params listed at https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/127893680/CORTX+Durability+Configuration.